packages/client-sdk-nodejs/src/internal/grpc/idle-grpc-client-wrapper.ts

```diff
@@ -53,6 +57,23 @@ export class IdleGrpcClientWrapper<T extends CloseableGrpcClient>
       this.client.close();
       this.client = this.clientFactoryFn();
     }
+
+    if (
+      this.reinitializeClientEveryInterval !== undefined &&
+      this.reinitializeClientEveryInterval
+    ) {
+      if (Date.now() - this.clientCreatedTime > 240000) {
+        this.logger.trace(
+          `Checking to see if client was created more than ${this.maxIdleMillis} ms`
+        );
+        this.logger.info(
+          'Client was created more than 4 minutes ago; reconnecting.'
+        );
+        this.client.close();
+        this.client = this.clientFactoryFn();
+        this.clientCreatedTime = Date.now();
+      }
+    }
```

Comment:
would prefer to add this stuff in a separate PR when its configuration options are added